funding/batch.go

```diff
@@ -230,19 +230,31 @@ func (b *Batcher) BatchFund(ctx context.Context,
 				err)
 		}
 
+		//nolint:lll
 		fundingReq, err := b.cfg.RequestParser(&lnrpc.OpenChannelRequest{
-			SatPerVbyte:        uint64(req.SatPerVbyte),
-			NodePubkey:         rpcChannel.NodePubkey,
-			LocalFundingAmount: rpcChannel.LocalFundingAmount,
-			PushSat:            rpcChannel.PushSat,
-			TargetConf:         req.TargetConf,
-			Private:            rpcChannel.Private,
-			MinHtlcMsat:        rpcChannel.MinHtlcMsat,
-			RemoteCsvDelay:     rpcChannel.RemoteCsvDelay,
-			MinConfs:           req.MinConfs,
-			SpendUnconfirmed:   req.SpendUnconfirmed,
-			CloseAddress:       rpcChannel.CloseAddress,
-			CommitmentType:     rpcChannel.CommitmentType,
+			SatPerVbyte:                uint64(req.SatPerVbyte),
```

Comment:
Was there a reason for re-ordering the fields? For the review it would've been a bit easier if the new fields were just added below.

Reply:
Yes, I reordered the fields starting with the general funding transaction params in `req` followed by the specific channel params in `rpcChannel`. Because these params are split the same way between the `BatchOpenChannelRequest` and `BatchOpenChannel` rpc messages.